Sources/ProjectDescription/InfoPlist.swift

```diff
@@ -42,7 +52,15 @@ public enum InfoPlist: Codable {
         switch self {
         case let .file(path):
             try container.encode("file", forKey: .type)
-            try container.encode(path, forKey: .path)
+            try container.encode(path, forKey: .value)
+        case let .dictionary(dictionary):
+            try container.encode("dictionary", forKey: .type)
+
+            // Codable doesn't support encoding values of type [String: Any]
+            // Until Swift supports it, we workaround it by serializing the value into a JSON string and then encoding it.
+            // https://bugs.swift.org/browse/SR-7788
+            let data = try JSONSerialization.data(withJSONObject: dictionary, options: [])
+            try container.encode(data.base64EncodedString(), forKey: .value)
```

Comment:
@tuist/core couldn't find a way to encode `[String: Any]` so I ended up doing this. Not quite happy about this because the command that prints the manifest to the console will show the base64 string. Ideas?